resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
```

Comment:
OK - thats another MAINT Fix that I need to push up on the Vanagon Side - will be having BJ with @McdonaldSeanp shortly so will discuss with him as it impacts his mods too.